elements/additionalpower.lua

```diff
@@ -1,31 +1,32 @@
 --[[
 # Element: Additional Power Bar
 
-Handles updating and visibility of a status bar displaying the player's additional power, such as Mana for Balance
+Handles updating and visibility of a status bar that displays player's additional power, such as Mana for Balance
 druids.
 
 ## Widget
 
-AdditionalPower - A StatusBar to represent the player's current additional power.
+AdditionalPower - A StatusBar that is used to represent player's additional power.
```

Comment:
"A StatusBar that is used to display the player's additional power." would be better imho...